### DISCUSSION TITLE
各位，小米多久更新一次题目啊

### DISCUSSION THREAD

**Discussion by Jonah-77** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#17] [Q&A] [answered]
不会按题库点进去刚好对不上吧，那就太惨了，试着答了一次，题库搜都来不及

**Comment by MlgmXyysd** [comment]
大概是每天上午下午都更，周末不更，我晚上趁他们下班才更新题库

**Reply by Jonah-77** [reply]
感谢大佬了，那我就等晚上大佬更新完题库我再答题，夜袭行动，可恶的小米 ![bc32670e-1f05-4f31-95e6-155e0d5a2b41.png](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/59916652/accbd53e-f4fe-4fd8-a051-f5026b9c5a37)

**Reply by flymeqy** [reply]
现在题目顺序跟题目正确错误都会随机

**Reply by Jonah-77** [reply]
大佬，他这个答题时间是什么时候，我点进去提示答题时间还没有开始

**Reply by MlgmXyysd** [reply]
最新公告，每日上午10点到下午18点

**Reply by Jonah-77** [reply]
> 最新公告，每日上午10点到下午18点

要求越来越多了

**Reply by Jonah-77** [reply]
> 最新公告，每日上午10点到下午18点

大佬，请教一下点击开始答题进去显示“问卷已提交”字样是什么情况呢？

**Reply by MlgmXyysd** [reply]
答题后7天内不可再次答题

**Reply by Jonah-77** [reply]
> 答题后7天内不可再次答题

很奇怪的是我第一次答题结束就是这样，别人的都是显示的分数

**Reply by MlgmXyysd** [reply]
不排除是出bug了，社区页面也看不到分数吗

**Reply by Jonah-77** [reply]
> 不排除是出bug了，社区页面也看不到分数吗

社区还有哪个地方可以看，我是从社区的BL解锁申请页面进入的，就是下面这个页面![Screenshot_2023-12-19-19-25-11-995_com.xiaomi.vipaccount-edit.jpg](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/59916652/e6eba4c4-a3de-4da7-b5aa-4acdd55ee2f5)

**Reply by MlgmXyysd** [reply]
点击 去答题 可以看到

**Reply by Jonah-77** [reply]
> 点击 去答题 可以看到

就是从这里点击“去答题”会出来页面然后再点击“开始答题”就出来这样![Screenshot_2023-12-19-13-18-18-702_com.xiaomi.vipaccount.jpg](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/59916652/ab7d77f4-4e20-4797-95da-4aa8fa2d3926)

**Comment by xbn2002** [comment, marked as answer]
最新：一周换一次题
